demisto_sdk/commands/validate/sdk_validation_config.toml

```diff
@@ -204,6 +205,7 @@ select = [
   "GR103",
   "GR104",
   "GR105",
+  "GR107",
```

Comment:
should we add it here, since it's removed from the legacy validate? @YuvHayun 
(see line 435 as well)

Reply:
Nope, it should only run with -g

Reply:
```suggestion
```